### ISSUE TITLE
Dashboard Profile tab is empty

### ISSUE THREAD

**Issue by Gohsato** [dask/distributed#3287]
Using `distributed==2.8.1, dask==2.8.1`

repro:
1. start a local cluster using `Client()`
2. open the dashboard from provided link
3. click profile tab
4. page is blank

The other tabs work as expected.
And the profile tab works when I downgrade to `distributed==2.8.0`

The chrome console also outputs this
```
Error rendering Bokeh items: TypeError: Cannot read property '0' of null
```

**Comment by jacobtomlinson** [comment]
Thanks for raising this. I'm also seeing this behaviour. I'll dig into it.

**Comment by jacobtomlinson** [comment]
When testing from the master branch this seems to be working again. Looks like it was fixed in #3277. Will be in the next release.

In the mean time please use `2.8.0` or you can install from the master branch (`pip install git+https://github.com/dask/distributed.git`).
